recipes/pytest-selenium/meta.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: pytest-selenium
+  version: 1.6.0
+
+source:
+  git_rev: v1.6.0
+  git_url: https://github.com/pytest-dev/pytest-selenium.git
+
+build:
+  noarch_python: True
```

Comment:
Not sure if you should use `noarch_python` on `conda-forge` recipes, even for pure Python packages...